### PR TITLE
PhpUnitNoExpectationAnnotationFixer - annotation in single line doc comment

### PR DESCRIPTION
**Pull request by kubawerlos**

### CODE REVIEW

File: src/Fixer/PhpUnit/PhpUnitNoExpectationAnnotationFixer.php [hunk 1]
```diff
@@ -265,7 +269,7 @@ private function extractContentFromAnnotation(Annotation $annotation)
             return '';
         }
 
-        $content = $matches[1];
+        $content = Preg::replace('/\*+\/$/', '', $matches[1]);
```
Comment (GrahamCampbell) [inline src/Fixer/PhpUnit/PhpUnitNoExpectationAnnotationFixer.php:272]: You need to match exactly 2 stars followed by a space. One star is not phpdoc, it's a comment.

Comment (kubawerlos) [reply src/Fixer/PhpUnit/PhpUnitNoExpectationAnnotationFixer.php:272]: That applies to the beginning of the comment, this RegEx is to match at least one `*`, followed by `$` at the end of the comment.

Comment (GrahamCampbell) [reply src/Fixer/PhpUnit/PhpUnitNoExpectationAnnotationFixer.php:272]: Ah, yes, I misread those slashes at the start.

Comment (kubawerlos) [reply src/Fixer/PhpUnit/PhpUnitNoExpectationAnnotationFixer.php:272]: Anything else regarding reviewing this then?

**Comment by SpacePossum** [comment]
Thank you @kubawerlos.

**Pull request merged**